docs/content/integrations/dbt/reference.mdx

```diff
@@ -131,62 +133,179 @@ Refer to the [Schedule documentation](/concepts/partitions-schedules-sensors/sch
+    yield from dbt.cli(["build"], context=context).stream()
 ```
 
-**Note**: The `key_prefix` argument only applies to assets loaded from this method. If you want to apply a prefix to the source keys that Dagster generates, pass in a `source_key_prefix` argument:
+### Customizing group names
+
+For dbt models, seeds, and snapshots, the default Dagster group name will be the [dbt group](https://docs.getdbt.com/docs/build/groups) defined for that node.
+
+| dbt node type         | dbt group name | Resulting Dagster group name |
+| --------------------- | -------------- | ---------------------------- |
+| model, seed, snapshot | `GROUP_NAME`   | `GROUP_NAME`                 |
+|                       | `None`         | `None`                       |
+|                       | finance        | finance                      |
+
+There are two ways to customize the asset keys generated by Dagster for dbt assets:
+
+1. Defining [meta config](https://docs.getdbt.com/reference/resource-configs/meta) on your dbt node, or
+2. Overriding Dagster's group name generation by implementing a custom `DagsterDbtTranslator`
+
+To override the group name generated by Dagster for a dbt node, you can define a `meta` key in your dbt project file, on your dbt node's property file, or on the node's in-file config block. The following example overrides the Dagster group name for the following model as `marketing`:
+
+```yaml
+models:
+  - name: customers
+    config:
+      meta:
+        dagster:
+          group: marketing
+```
+
+Alternatively, you can implement a custom `DagsterDbtTranslator` to override the Dagster group name generation for all dbt nodes in your dbt project. The following example defines `snowflake` as the group name for all dbt nodes:
+
+```python startafter=start_custom_asset_key_dagster_dbt_translator endbefore=end_custom_asset_key_dagster_dbt_translator file=/integrations/dbt/dbt.py dedent=4
+from pathlib import Path
+from dagster import AssetKey, OpExecutionContext
+from dagster_dbt import DagsterDbtTranslator, DbtCliResource, dbt_assets
+from typing import Any, Mapping
+
+manifest_path = Path("path/to/dbt_project/target/manifest.json")
+
+class CustomDagsterDbtTranslator(DagsterDbtTranslator):
+    def get_asset_key(self, dbt_resource_props: Mapping[str, Any]) -> AssetKey:
+        return self.get_asset_key(dbt_resource_props).with_prefix("snowflake")
+
+@dbt_assets(
+    manifest=manifest_path,
+    dagster_dbt_translator=CustomDagsterDbtTranslator(),
+)
+def my_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource):
+    yield from dbt.cli(["build"], context=context).stream()
+```
+
+### Customizing descriptions
+
+For dbt models, seeds, and snapshots, the default Dagster description will be the dbt node's description.
+
+There is one way to customize the description generated by Dagster for dbt assets:
+
+1. Overriding Dagster's description generation by implementing a custom `DagsterDbtTranslator`
```

Comment:
Consider:

"For dbt models, seeds, and snapshots, the default Dagster description will be the dbt node's description.

To override the Dagster group name generation for all dbt nodes in your dbt project, you can implement a custom `DagsterDbtTranslator`..."

This format breaks the symmetry with other sections, but is more readable for someone who just cares about modifying the description